### PR TITLE
Certformat

### PR DESCRIPTION
**Pull request by MangyCoyote**
A bit of additional documentation for AKV getcertificate + added iTest for pfx file format.

### CODE REVIEW

File: Commands.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ If storage account has more than 5,000 blobs, it is required to use `--prefix`,
 
 |**command**|description|
 |-|-|
-|**`azmi getcertificate`**|Downloads single certificate from Azure Key Vault.|
+|**`azmi getcertificate`**|Downloads certificate (PEM of PFX format; certificate(s) and private key bundle) from Azure Key Vault.|
```
Comment (iricigor) [inline Commands.md:124]: Hm, lets move this additional text to explanation section?

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ azmi setblob --file ~/info.txt --blob $CONTAINER_URL/myhostname/info.txt --ident
 azmi getsecret --secret $SECRET_URL
 azmi getsecret --secret $SECRET_URL/$VERSION
 
-# fetch latest or specific version of a certificate from Azure Key Vault
+# fetch latest or specific version of a certificate (PEM or PFX format; certificate(s) and private key bundle) from Azure Key Vault
```
Comment (iricigor) [inline README.md:34]: Again, I think this explanation is too big. One will just download the certificate in the same format as it is uploaded, right?

File: Commands.md (hunk 1 above)
Comment (MangyCoyote) [reply Commands.md]: Okay, did a bit redo.

File: README.md (hunk 2 above)
Comment (MangyCoyote) [reply README.md]: Okay, did a bit redo.

**Pull request merged**